src/app/components/SearchResultsPage/SearchResultsPage.jsx

```diff
@@ -57,23 +57,37 @@ const SearchResultsPage = (props, context) => {
       <main className="main-page">
         <div className="nypl-page-header">
           <div className="nypl-full-width-wrapper">
-            {breadcrumbs}
-            <h2 aria-label={h2Label}>
-              Search results
-            </h2>
-            <Search
-              searchKeywords={searchKeywords}
-              field={field}
-              spinning={spinning}
-              createAPIQuery={createAPIQuery}
-            />
+            <div className="nypl-column-three-quarters">
+              {breadcrumbs}
+              <h2 aria-label={h2Label}>
+                New York Public Library Research Catalog
+              </h2>
+              <Search
+                searchKeywords={searchKeywords}
+                field={field}
+                spinning={spinning}
+                createAPIQuery={createAPIQuery}
+              />
+
+              {
+                !!(totalResults && totalResults !== 0) && (
+                  <Sorter
+                    sortBy={sortBy}
+                    page={page}
+                    searchKeywords={searchKeywords}
+                    createAPIQuery={createAPIQuery}
+                  />
+                )
+              }
+            </div>
           </div>
         </div>
+
         <div className="nypl-full-width-wrapper">
 
           <div className="nypl-row">
             <div
-              className="nypl-column-full"
+              className="nypl-column-three-quarters"
```

Comment:
@ktp242 check the PR's description. It has a link to the updated design and I'm basing the PR off that and it has `three-quarters`.